src/api/app/controllers/webui/request_controller.rb

```diff
@@ -209,12 +209,7 @@ def accept_request
 
   def forward_request_to(fwd)
     tgt_prj, tgt_pkg = params[fwd].split('_#_') # split off 'forward_' and split into project and package
-    description = @req.description
-    description ||= ""
-    who = @req.creator
-    description += ' (forwarded request %d from %s)' % [params[:number], who]
 
-    req = nil
```

Comment:
@mdeniz I was wrong. It is needed, otherwise even if `req` is set, it is just a local variable and then there are errors (that's the reason why the tests were failing). 

Although we still should check if it is possible that it is nil, if so then there is a bug, if not this method can be refactored.